python/chigger/observers/TimerObserver.py

```diff
@@ -31,7 +31,7 @@ def init(self, window):
         Add the TimerEvent for this object.
         """
         super(TimerObserver, self).init(window)
-        window.getVTKInteractor().AddObserver('TimerEvent', self._callback)
+        self._id = window.getVTKInteractor().AddObserver('TimerEvent', self._callback)
```

Comment:
I don't think this id is what is used in `HasObserver()`.
Doing a `window.getVTKInteractor().HasObserver(self._id)` right after this always seems to return False.